wwwroot/src/components/Header.js

```diff
@@ -86,73 +131,118 @@ export function Logo() {
   )
 }
 
-function HeaderButtons({location, signedIn}) {
+function NavbarButtons() {
   const classes = useStyles();
-
-  if (location.pathname==='/signin'||location.pathname==='/signup') {
-    return (<></>);
-  } else if (signedIn) {
-    return (<></>);
-  } else {
-    return (
-      <Box className={classes.buttonCollection} >
+  return (
+    <>
```

Comment:
Please use <Fragment> from react instead of the empty <>.

Reply:
<React.Fragment> ?

Reply:
Replaced all instances of empty <> with <React.Fragment>